### ISSUE TITLE
Option to add masks as input

### ISSUE THREAD

**Issue by jennyfolkesson** [czbiohub/microDL#69]
Some ground truth masks will be generated by some manual or semi-automatic means when a simple thresholding isn't enough.
microDL should be able to accept a mask directory as an input and add that to the training data seamlessly.

**Comment by jennyfolkesson** [comment]
Fixed with PR https://github.com/czbiohub/microDL/pull/77
